lbrynet/extras/daemon/Daemon.py

```diff
@@ -2323,66 +2180,42 @@ def jsonrpc_claim_list_mine(self, account_id=None, page=None, page_size=None):
         )
 
     @requires(WALLET_COMPONENT)
-    async def jsonrpc_claim_list(self, name):
+    async def jsonrpc_claim_search(self, name, channel_id=None, winning=False):
         """
-        List current claims and information about them for a given name
+        Search for claims on the blockchain.
 
         Usage:
-            claim_list (<name> | --name=<name>)
+            claim_search (<name> | --name=<name>) [--channel_id=<channel_id>] [--winning]
```

Comment:
when would `winning` be used instead of `resolve(<name>)`? is `winning` needed?

Reply:
This is for paginating a huge channel. `resolve` will only return a small sampling of the latest claims in a channel. So for home page user would still do `resolve` but if they are on a screen that is showing all of the claims in a channel with pagination they would use claim search which will support filtering and better search functionality in the future.

Reply:
When would they want non winning claims when listing those in a channel? What's the use-case for a channel owner to have two competing claims for the same name?

Reply:
Perhaps a very large channel with many contributors and if one group in the organization doesn't agree with another group in the organization and they fight over who's claim will show up. Both having the channel key to publish....